azurerm/config.go

```diff
@@ -472,6 +473,13 @@ func (c *ArmClient) registerAutomationClients(endpoint, subscriptionId string, a
 	scheduleClient := automation.NewScheduleClientWithBaseURI(endpoint, subscriptionId)
 	c.configureClient(&scheduleClient.Client, auth)
 	c.automationScheduleClient = scheduleClient
+
+	runbookDraftClient := automation.NewRunbookDraftClientWithBaseURI(endpoint, subscriptionId)
+	setUserAgent(&runbookDraftClient.Client)
+	runbookDraftClient.Authorizer = auth
+	runbookDraftClient.Sender = sender
+	runbookDraftClient.SkipResourceProviderRegistration = c.skipProviderRegistration
+	c.automationRunbookDraftClient = runbookDraftClient
```

Comment:
can we switch this over to using `c.configureClient(&runbookDraftClient.Client, auth)`